functions/go/list-setters/README.md

```diff
@@ -0,0 +1,80 @@
+# list-setters
+
+## Overview
+
+<!--mdtogo:Short-->
+
+Lists information about [setters] like setter name, values and count.
+
+Refer to the [create-setters] function documentation for information about creating new setters or [apply-setters] function documentation for information about parameterizing field values using setters.
```

Comment:
information about updating the field values parameterized by setters.

Reply:
done